editor/layout/style.scss

```diff
@@ -22,6 +22,11 @@
 	overflow-y: auto;
 }
 
+.mobile .editor-layout__editor {
+	overflow-y: scroll;
+	-webkit-overflow-scrolling: touch;
+}
```

Comment:
[MDN seems very wary](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling) of using `-webkit-overflow-scrolling`. Are we safe? Is there a risk of targeting unwanted platforms where this could have a bad impact? We could target this to iOS.

Reply:
@mcsf only webkit should interpret this from the vendor prefix and body will have the `.mobile` class on for mobile devices? Have tested on iPad and Nexus tablet.

Reply:
> only webkit should interpret this

Indeed, but my concern was whether this could affect non-iOS WebKit agents negatively. However, if you've tested on a Nexus and feel confident about this, I can withdraw my question.

Reply:
Should we really target `.mobile` here? What is the connection between the two? Can we not just add it to `.editor-layout__editor`?